app/main.py

```diff
@@ -1,3 +1,6 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    count = 0
+    for i in phrase:
+        if i.lower() == letter.lower():
+            count += 1
+    return count
```

Comment:
avoid using loops to solve this task. Instead, consider utilizing Python's built-in string functions to simplify the solution.